### PR TITLE
Rework discord webhook

### PR DESCRIPTION
**Pull request by leduythuccs**
- If the webhook link is not found, it will try to get the default link from `DISCORD_WEBHOOK`.
- Add on_new_contest event
- Show more information for on_new_problem
- Refactor code
- Send webhook on new organization problem

image

### CODE REVIEW

File: judge/tasks/webhook.py [hunk 1]
```diff
@@ -130,17 +129,43 @@ def on_new_tag(problem_code, tag_list):
...
-        description=description,
-        color='03b2f8',
-    )
+    send_webhook(webhook, f'New tag added for problem {url}', description, None)
 
-    webhook.add_embed(embed)
-    webhook.execute()
+
+@shared_task
+def on_new_contest(contest_key, is_suggested=False):
```
Comment (hieplpvip) [inline judge/tasks/webhook.py:141]: Delete `is_suggested`

**Comment by hieplpvip** [comment]
Can't test this but it looks good to me. Merge as you see fit.

File: judge/tasks/webhook.py (hunk 1 above)
Comment (leduythuccs) [reply judge/tasks/webhook.py:141]: w..w..w..w..w..why

Comment (hieplpvip) [reply judge/tasks/webhook.py:141]: It looks like a leftover from copying `on_new_problem`. You also don't use it in the code.

Comment (leduythuccs) [reply judge/tasks/webhook.py:141]: oh you r right, my bad

Comment (leduythuccs) [reply judge/tasks/webhook.py]: Fixed in new commit

**Review by hung3a8 — APPROVED** [review]
Lgtm

**Pull request merged**